CONTRIBUTING.md

```diff
@@ -352,7 +354,44 @@ In cases where a PR is not worth noting in the release notes, you can also tell
 
 Finally, don't fret about this too much! The Release Drafter configuration and labeling scheme may take some time to fine-tune, and the drafted release notes can always be manually edited before final publication.
 
-## Publishing to npm
+## npm package
+
+### Testing the package locally
+
+1. Go to the root of your local working copy of explorer-1
+   ```bash
+   $ cd path/to/explorer-1
+   ```
+2. Create a symlink from your global `node_modules` directory to the local explorer-1 directory
+   ```bash
+   $ npm link
+   ```
+3. In the root of the project you want to test explorer-1 in, add the symlink. This works even if you had previously installed the production version of explorer-1.
```

Comment:
This could maybe be formatted better, but I thought a little more detail could be useful.

```suggestion
3. In the root* of the project you want to test explorer-1 in, add a symlink from the project's `node_modules` to your global `node_modules`. This results in a two-part symlink chain: Your project's `node_modules/@nasa-jpl/explorer-1` ➡️ npm's global `node_modules/@nasa-jpl/explorer-1` ➡️ your cloned explorer-1 repo. This works even if you had previously installed the production version of explorer-1.

   _* Note: Currenly in WCP, you must go into `cms/theme/static-src` before running this command._
```